src/main/resources/db/changelog/db.changelog-RDM-9910.xml

```diff
@@ -0,0 +1,28 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="rdm-9910" runInTransaction="false" author="daniel.lysiak1@hmcts.net">
+        <sql dbms="postgresql"
+            endDelimiter="\nGO"
+            splitStatements="true"
+            stripComments="true">
+            ALTER TABLE case_data ADD ls_processed boolean DEFAULT false;
+
+            UPDATE case_data SET ls_processed = true;
```

Comment:
we need to double check that we don't risk LS ignoring some cases not yet indexed by doing this right?